### PR TITLE
Mds 416:- updates to doc sand changed code so responses match 

### PR DESCRIPTION
**Pull request by parkeradam**

### CODE REVIEW

File: resources/public/api/conf/2.0/schemas/income-sa-additional-information-response.json [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
         "self": {
           "type": "object",
-          "description": "The URI to this resource",
+          "description": "The Uniform Resource Identifier (URI) for this endpoint.",
           "properties": {
             "href": {
               "type": "string",
               "description": "The URL of the endpoint.",
-              "example": "/individuals/income/sa/additional-information?matchId=57072660-1df9-4aeb-b4ea-cd2d7f96e430{&fromDate,toDate}"
+              "example": "/individuals/income/sa/additional-information?matchId=57072660-1df9-4aeb-b4ea-cd2d7f96e430{&fromTaxYear,toTaxYear}"
```
Comment (ajchilts) [inline resources/public/api/conf/2.0/schemas/income-sa-additional-information-response.json:17]: Did we say we should include the date itself in schemas? Other than root HATOES links? 

https://github.com/hmrc/individuals-employments-api/blob/master/resources/public/api/conf/2.0/schemas/employments-response.json#L17

Example from employments ^^

File: resources/public/api/conf/2.0/schemas/income-sa-foreign-response.json [hunk 2]
```diff
@@ -9,12 +9,12 @@
...
         "self": {
           "type": "object",
-          "description": "The URI to this resource",
+          "description": "The Uniform Resource Identifier (URI) for this endpoint.",
           "properties": {
             "href": {
               "type": "string",
               "description": "The URL of the endpoint.",
-              "example": "/individuals/income/sa/foreign?matchId=57072660-1df9-4aeb-b4ea-cd2d7f96e430{&fromDate,toDate}"
+              "example": "/individuals/income/sa/foreign?matchId=57072660-1df9-4aeb-b4ea-cd2d7f96e430{&fromTaxYear,toTaxYear}"
```
Comment (ajchilts) [inline resources/public/api/conf/2.0/schemas/income-sa-foreign-response.json:17]: Actual dates in schemas?

File: resources/public/api/conf/2.0/schemas/income-sa-additional-information-response.json (hunk 1 above)
Comment (owen-opencast) [reply resources/public/api/conf/2.0/schemas/income-sa-additional-information-response.json:17]: 👍

**Review by owen-opencast — COMMENTED** [review]
Looks good - just need @ajchilts comments addressing

**Review by ajchilts — APPROVED** [review]

**Pull request merged**